WordPress/gutenberg
# Button: refactor Storybook to controls and align docs

## What?
PR adds `isBusy` state knobs to Button component stories.

## Testing Instructions
1. Run `npm run storybook:dev`
2. Check button stories for `primary`, `secondary` or `isDestructive `.

## Screenshots or screencast 

![CleanShot 2022-09-13 at 10 17 49](https://user-images.githubusercontent.com/240569/189825605-2cd519d6-aff9-41e4-8c63-f6a8906d0608.png)
